lib/grape/validations/params_scope.rb

```diff
@@ -232,16 +232,25 @@ def validates(attrs, validations)
         default = validations[:default]
         doc_attrs[:default] = default if validations.key?(:default)
 
-        values = options_key?(:values, :value, validations) ? validations[:values][:value] : validations[:values]
+        values = validations[:values]
+        if values.is_a? Hash
+          excepts = values[:except]
+          values = values[:value]
+          values = nil if values.is_a? Proc
```

Comment:
I should add that, tbh, I'm not a fan of the current syntax for except.  After playing around with it, I think we'd be better served by a separate top level validator to handle value exclusion. But in the mean time, the current syntax should at least work properly.  ;)